tests/memlimit/dockerfile/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:bionic
+FROM ubuntu:focal
```

Comment:
Since this is a test I think it can stay unchanged, especially if we're not dropping bionic.